test/parallel/test-tls-get-ca-certificates-bundled.js

```diff
@@ -9,12 +8,9 @@ const assert = require('assert');
 const tls = require('tls');
 const { assertIsCAArray } = require('../common/tls');
 
-const certs = tls.getCACertificates('bundled');
+const certs = tls.getCACertificates({ type: 'bundled', format: 'string' });
```

Comment:
```suggestion
const certs = tls.getCACertificates('bundled');
assert.deepStrictEqual(certs, tls.getCACertificates({ type: 'bundled', format: 'pem' }));
```

It would be good to keep the string argument tests, since they still verify expected behavior. I suggest adding a check to show the options form returns the same result.

Reply:
'pem' is not a valid format (only 'string', 'buffer', 'x509' are supported). I updated the test to keep the shorthand form and add the options form with format: 'string'.

Reply:
The point I want to clarify: the three modified tests (test/parallel/test-tls-get-ca-certificates-bundled.js, ...-system.js, ...-default.js) must still cover `undefined` and string arguments (`bundled`, `system`, `default`). Changing them to only test the Object form drops necessary coverage. 🙂

Reply:
I’ve added clarifications as suggested!!